python/cudf/cudf/core/dataframe.py

```diff
@@ -3158,20 +3159,6 @@ def take(self, positions, keep_index=True):
         out.columns = self.columns
         return out
 
-    def __copy__(self):
```

Comment:
Similar to above, do we not use these anymore/anywhere?

Reply:
These aren't necessary because the logic in the `Serializable` class is sufficient to handle copying (the serialization/deserialization process always results in deep copies). There may be performance reasons to prefer overriding `__copy__` at this more precise level, but if so I'd rather do that consistently for all `Frame` types at a later point when it's easier to get consistent benchmarks across this family of classes.

Reply:
ah - it's in `Serializable`. 👍